pkg/kubelet/dockertools/manager.go

```diff
@@ -1896,6 +1895,7 @@ func (dm *DockerManager) SyncPod(pod *api.Pod, runningPod kubecontainer.Pod, pod
 		pod.Status.PodIP = dm.determineContainerIP(pod.Name, pod.Namespace, podInfraContainer)
 	}
 
+	containerStarted := 0
```

Comment:
(nit) prefer plural: containersStarted
(containerStarted sounds like a bool flag to me, not a count)